crates/executor/src/transaction.rs

```diff
@@ -169,19 +169,30 @@ where
     let (l2_gas_limit, mut tx_info, tx_state) =
         match simulate_transaction(tx, tx_index, state, block_context) {
             Ok((tx_info, tx_state)) => {
+                metrics::increment_counter!(
+                    "l2_gas_accounting.gas_limit_search.without_binary_search"
```

Comment:
I'm not sure these names are quite right.

What about using something that explicitly states that this is _fee estimation_?

Something along the lines of `rpc_fee_estimation.without_binary_search` maybe? I'm open to alternative suggestions.